### PR TITLE
Don't output the LR warning when finetuning.

### PR DESCRIPTION
**Pull request by stephenroller**
**Patch description**
Previously, when you started fine tuning a model, the LR scheduler would reset. But it would also show a weird cryptic message when it reset.

Now, this cryptic message only shows when the LR scheduler actually changed; not when the user explicitly knows they're fine tuning.

### CODE REVIEW

File: parlai/core/torch_agent.py [hunk 1]
```diff
@@ -769,7 +769,14 @@ def _invsqrt_lr(step):
             # first make sure there are no null pointers
             states = {}
 
-        if states and states.get('lr_scheduler_type') != self.opt['lr_scheduler']:
+        if (
+            # there is already an old LR scheduler saved on disk
+            states and
+            # and the old LR scheduler is different
+            states.get('lr_scheduler_type') != self.opt['lr_scheduler'] and
```
Comment (emilydinan) [inline parlai/core/torch_agent.py]: only question... do we need an opt.get('lr_scheduler') here?

Comment (stephenroller) [reply parlai/core/torch_agent.py]: A super old model might have been trained before we introduce lr_schedulers.

Comment (emilydinan) [reply parlai/core/torch_agent.py]: then why didn't you put an opt.get there?

Comment (stephenroller) [reply parlai/core/torch_agent.py]: Hmm, right above it will `raise ValueError` if it's not in the dictionary so i guess it's handled
